cluster/addons/addon-manager/Makefile

```diff
@@ -18,7 +18,7 @@ TEMP_DIR:=$(shell mktemp -d)
 VERSION=v9.1.4
```

Comment:
Can you bump this version to v9.1.5 as well?

Reply:
Whoops, done